packages/plazo_mx/presentation/lib/features/login/bloc/provider.dart

```diff
@@ -0,0 +1,22 @@
+
+
+import 'package:domain/features/login/repository/validate_impl.dart';
+import 'package:flutter/material.dart';
+
+
+//! scoped instance
+
+class Provider extends InheritedWidget{
```

Comment:
?

Reply:
here I used my own provider with inheredWidget
yes I could be single global instance like(final RegistrationImpl _registrationImpl = RegistrationImpl();) for this app but best way to use scoped instance for large projects, that s way inheredWidget allow us to do it